TaskHelper performance optimization

yamiYori opened the pull request:
### Change
- Replace int `MaxTask` with bool `hasPendingTask`
- Fast return in `TaskHelper.ProcessNextTask()`

### Note

由于MaxTask的可见性是public，可能要看下是否有外部模块依赖该成员

yamiYori committed:
TaskHelper performance optimization
- Replace `MaxTask` with bool `hasPendingTask`
- Fast return in `TaskHelper.ProcessNextTask()`

AtmoOmen commented on Helpers/TaskHelper/TaskHelper.cs:
```diff
@@ -20,7 +20,7 @@ public TaskHelper()
     public  int                        NumQueuedTasks => Queues.Sum(q => q.Tasks.Count) + (CurrentTask == null ? 0 : 1);
     public  bool                       IsBusy => CurrentTask != null || Queues.Any(q => q.Tasks.Count > 0);
-    public  int                        MaxTasks        { get; private set; }
+    public  bool                       hasPendingTask  { get; private set; } = false;
```
Field name should be capitalized
Besides, I suggest this to become a private field

yamiYori replied:
Fixed.

yamiYori committed:
Code review fix
- Correct name casing of `hasPendingTask`
- Change `HasPendingTask` to a private field
